extensions/labgraph_audiogen/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='labgraph_audiogen',
+    version='0.1',
+    description="Audio generation on labgraph",
+    packages=find_packages(),
+    install_requires=[
+        'Click',
```

Comment:
Can you add the library versions which you are using? And add information on which os system it's tested?

Reply:
- Added the versions tested
- Added initial O.S Support information on the ReadME